administrator-manual/en/webtop5.rst

```diff
@@ -412,6 +495,27 @@ that can be used within the parameterized fields within the domain mailcard edit
 
 In this way it is possible to create a single mailcard that will be automatically customized for every user who does not use his own mailcard.
 
+Configure multiple mailcards for a single user
+==============================================
+
+It is possible to configure multiple mailcards (html signatures) for each individual user.
```

Comment:
"html" is an abbreviation, it should be uppercase: "HTML"

Reply:
done